#!/usr/bin/env python
PACKAGE = "controls"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()



gen.add("a_factor", double_t, 0, "Glideslope : a", .01,  .005, .1)
gen.add("v_max", double_t, 0, "Glideslope : v_max", .02,  -.1, .2)
gen.add("starting", bool_t, 0, "Glideslope : starting", False)


exit(gen.generate(PACKAGE, "glideslope", "glideslope"))
